src/WorkflowCore/Models/WorkflowOptions.cs

```diff
@@ -26,6 +27,10 @@ public WorkflowOptions(IServiceCollection services)
             PollInterval = TimeSpan.FromSeconds(10);
             IdleTime = TimeSpan.FromMilliseconds(100);
             ErrorRetryInterval = TimeSpan.FromSeconds(60);
+            JsonSerializerSettings = new JsonSerializerSettings
```

Comment:
I think this belongs on the settings for a specific persistence provider that requires JSON, rather than on the global workflow settings